NOTICE.txt

```diff
@@ -2871,6 +2855,19 @@ SOFTWARE.
 
 ---
 
+## semver
+
+This product contains 'semver' by Semver.
+
+"Semantic Versioning" or "SemVer" contain a set of rules and requirements that dictate how version numbers are assigned and incremented
+
+* HOMEPAGE:
+  * https://github.com/semver/semver
+
+* LICENSE: MIT
```

Comment:
@amyblais I've looked at this again, and I don't see any dependency in our package.lock.json on this repo. Are you sure it isn't `node-semver` (i.e. the one that is *removed* immediately above this block in the diff?

Reply:
I can see it here: https://github.com/mattermost/mattermost-webapp/blob/release-5.25/package.json#L73. Not sure if I should just keep node-semver.

Reply:
That's a package in npmjs called `semver`: https://www.npmjs.com/package/semver - if you look at the link to Github from there, it goes to github.com/npm/node-semver , not https://github.com/semver/semver

Reply:
Ok, removed ``semver`` and added back ``node-semver``.